lib/logstash/inputs/beats.rb

```diff
@@ -175,6 +177,7 @@ def register
   def create_server
     server = org.logstash.beats.Server.new(@host, @port, @client_inactivity_timeout)
     if @ssl
+      check_no_exec_mount!
```

Comment:
Also, Netty has this https://netty.io/4.0/api/io/netty/handler/ssl/OpenSsl.html#isAvailable()

We can check OpenSsl.isAvailable(), or if we want the exception, call `OpenSsl.ensureAvailability()`. I have tried this in `tealess` and it fails when /tmp is noexec (or if the .so library fails to load in any way)